engine/gameobject/src/gameobject/comp_script.cpp

```diff
@@ -502,8 +502,8 @@ namespace dmGameObject
                 lua_settable(L, -3);
 
                 lua_pushliteral(L, "gamepad_buttons");
-                lua_createtable(L, dmHID::MAX_GAMEPAD_AXIS_COUNT, 0);
-                for (int i = 0; i < dmHID::MAX_GAMEPAD_AXIS_COUNT; ++i)
+                lua_createtable(L, dmHID::MAX_GAMEPAD_BUTTON_COUNT, 0);
```

Comment:
Unrelated but discovered while debugging.

Surely this must have been wrong? Both MAX_GAMEPAD_AXIS_COUNT and MAX_GAMEPAD_BUTTON_COUNT have the value 32 so it has so far not been a problem